### PR TITLE
build: minimum supported Nodejs version is 14.15.0

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

Already present.

### Motivation / Use-Case

Drop Nodejs v12. Prepare for next version.

### Breaking Changes

minimum supported Nodejs version is 14.15.0

### Additional Info
No

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = (api) => {
           modules: false,
           targets: {
             esmodules: true,
-            node: "0.12",
+            node: "14.15.0",
```
Comment (alexander-akait) [inline babel.config.js:14]: Keep this, it is for browsers, so let's support all of them

Comment (snitin315) [reply babel.config.js:14]:
```suggestion
            node: "0.12",
```

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
@snitin315 Can we fix in master tests? I think we need to use the same hack for all macos versions + update deps?

**Comment by alexander-akait** [comment]
@snitin315 Feel free to merge when CI will be green

**Pull request merged**